Enterprise-CMCS/macpro-mdct-mfp
# [MFP] WP - Initiatives Details - Close out functionality

### Description



### Related ticket(s)


MDCT-2998


### What this PR covers
- Initiative does in fact, close out and that information is saved and sent to the db (the only changes that are visible right now would show up in the network tab) 

### What this PR does not cover/yet to be worked on
- autosave & hydration :) [CMDCT-194](https://jiraent.cms.gov/browse/CMDCT-194)
- disabled close out button + return to dashboard button [CMDCT-201](https://jiraent.cms.gov/browse/CMDCT-201)
- Pressing "Yes, close out initiative" or "Save and return" takes user back to the initiative dashboard [CMDCT-204](https://jiraent.cms.gov/browse/CMDCT-204)

### What I'd like feedback on
- Am I sending the correct data to the db? Did I implement this correctly (is there any risk of my code overwriting data that it's not supposed to)? Is there anything I'm missing that would be helpful for statusing/accessing initiatives in SAR?

---
### How to test

- Confirm that on closing out the initiative, correct fieldData + isInitiativeClosed=true is submitted to the db
- Confirm that on Save & Return, correct fieldData is submitted to the db
- Confirm that fieldData is nested per initiative (create multiple initiatives, make sure that each initiative has correct field data)

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
